### ISSUE TITLE
Primitive type lack of target depends size_t and ssize_t, and I think intptr_t == ssize_t, uintptr_t=size_t, choose one?

### ISSUE THREAD

**Issue by lygstate** [vlang/v#4965]
```
bool

string

i8    i16  int  i64      i128 (soon)
byte  u16  u32  u64      u128 (soon)

rune // represents a Unicode code point

f32 f64

byteptr // these two are mostly used for C interop
voidptr

any // similar to C's void* and Go's interface{}

```
And also isize, usize for consistence?

**Comment by spytheman** [comment]
V has a builtin `size_t` type for C interoperability:
```v
fn main(){
        eprintln('size_t size: ${sizeof(size_t)}')
}
```
... which on my machine prints: `size_t size: 8`

**Comment by lygstate** [comment]
Does size_t are primitive type? or typedef for C?

**Comment by Gladear** [comment]
implemented in cc8ee5fb
